qirlib/src/generation/interop.rs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+pub use inkwell::IntPredicate;
```

Comment:
Might be better to redefine this instead of exposing the Inkwell type?

Reply:
@samarsha I agree, I we shouldn't reexport inkwell out of the interop layer